dev/NavigationView/NavigationView.xaml

```diff
@@ -363,7 +363,7 @@
                                                     <Style TargetType="FlyoutPresenter">
                                                         <Setter Property="Padding" Value="{ThemeResource TopNavigationViewOverflowMenuPadding}" />
                                                         <!-- Set negative top margin to make the flyout align exactly with the button -->
```

Comment:
Can you remove the comment that no longer applies.

Reply:
Done.